circuits/cpp/src/aztec3/constants.hpp

```diff
@@ -23,7 +23,7 @@ constexpr size_t KERNEL_NEW_NULLIFIERS_LENGTH = 4;
 constexpr size_t KERNEL_NEW_CONTRACTS_LENGTH = 1;
 constexpr size_t KERNEL_PRIVATE_CALL_STACK_LENGTH = 8;
 constexpr size_t KERNEL_PUBLIC_CALL_STACK_LENGTH = 8;
-constexpr size_t KERNEL_NEW_L2_TO_L1_MSGS_LENGTH = 4;
+constexpr size_t KERNEL_NEW_L2_TO_L1_MSGS_LENGTH = 2;
```

Comment:
In the same manner as `KERNEL_NEW_COMMITMENTS_LENGTH` etc. One kernel might have multiple inner function calls and such so it can be useful to allow kernels with more than individual calls. However, for now thought that it made sense that the values were the same similar to what it is for the commitments and nullifiers.